specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -51,6 +51,527 @@
         }
       }
     },
+    "AcsRouterJobCancelledEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobCancelled event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "note": {
+          "description": "Router Job Note",
+          "type": "string"
+        },
+        "dispositionCode": {
+          "description": "Router Job Disposition Code",
+          "type": "string"
+        }
+      }
+    },
+    "AcsRouterJobClassificationFailedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobClassificationFailed event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "classificationPolicyId": {
+          "description": "Router Job Classification Policy Id",
+          "type": "string"
+        },
+        "errors": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterCommunicationError"
+          }
+        }
+      }
+    },
+    "AcsRouterJobClassifiedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobClassified event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "queueInfo": {
+          "description": "Router Job Queue Info",
+          "$ref": "#/definitions/AcsRouterQueueInfo"
+        },
+        "classificationPolicyId": {
+          "description": "Router Job Classification Policy Id",
+          "type": "string"
+        },
+        "priority": {
+          "description": "Router Job Priority",
+          "type": "integer",
+          "format": "int32"
+        },
+        "attachedWorkerSelectors": {
+          "description": "Router Job Attached Worker Selector",
+          "type": "array",
+          "format": {
```

Comment:
This shouldn't have the format property because it is already using $ref.

Reply:
updated